runtime/node/package.json

```diff
@@ -19,6 +19,8 @@
     "adaptive-expressions": "4.12.0-rc3",
     "botbuilder": "4.12.0-rc3",
     "botbuilder-ai": "4.12.0-rc3",
+    "botbuilder-applicationinsights": "4.12.0-rc3",
```

Comment:
let's update this to the released version instead of rc.

Reply:
done.